packages/map/.gitignore

```diff
@@ -0,0 +1,4 @@
+styles.css
+lib
+test-report.log
+sample_data/local.jsx
```

Comment:
can we get the app to run without this file too? (so that it doesn't fail on first `npm start`)